File: Tareas_FdP/Tarea02vscode/Ejercicio03.cs
using System;

class Area_Cono {
    static void Main() {
        double radio, generatriz, altura, aBase, alado, aTotal, volumen;

        Console.WriteLine("Ingrese el radio del cono: ");
        radio = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Ingrese la generatriz del cono: ");
        generatriz = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Ingrese la altura del cono: ");
        altura = Convert.ToDouble(Console.ReadLine());

        aBase = Math.PI * (radio * radio);
        alado = Math.PI * radio * generatriz;
        aTotal = aBase + alado;
        volumen = (1.0 / 3.0) * aBase * altura;

        Console.WriteLine("El área de la base es: " + aBase);
        Console.WriteLine("El área lateral es: " + alado);
        Console.WriteLine("El área total es: " + aTotal);
        Console.WriteLine("El volumen es: " + volumen);
    }
}

File: Tareas_FdP/Tarea02vscode/Ejercicio03.py
import math

radio = 0.0
generatriz = 0.0
altura = 0.0
aBase = 0.0
alado = 0.0
aTotal = 0.0
volumen = 0.0

radio = float(input("Ingrese el radio del cono: "))
generatriz = float(input("Ingrese la generatriz del cono: "))
altura = float(input("Ingrese la altura del cono: "))

aBase = math.pi * (radio * radio)
alado = math.pi * radio * generatriz
aTotal = aBase + alado
volumen = (1 / 3) * aBase * altura

print("El área de la base es:", aBase)
print("El área lateral es:", alado)
print("El área total es:", aTotal)
print("El volumen es:", volumen)